 int32  Servo_ID
 int32  Present_Position
 int32  Present_Velocity
 int32  Present_PWM
 int32  Present_Load
